core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.ts

```diff
@@ -42,22 +50,29 @@ export class ExplorationFooterComponent {
   resizeSubscription!: Subscription;
   contributorNames: string[] = [];
   hintsAndSolutionsAreSupported: boolean = true;
+  numberofCheckpoints: number = 0;
```

Comment:
numberOfCheckpoints
```suggestion
  numberOfCheckpoints: number = 0;
```

Reply:
Actually, would `checkpointCount` be better?

Reply:
Ya, replaced.